src/@next/Pagination/IconOnlyPagination.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {
+  IconOnlyPaginationStyledNav,
+  StyledBorderPaginationButton,
+  StyledLabel,
+} from './PaginationStyle';
+import { IconOnlyPaginationProps } from './paginationUtils';
+import { Typography } from '../Typography';
+import { Icon } from '../Icon';
+
+export const IconOnlyPagination = React.forwardRef<
+  HTMLElement,
+  IconOnlyPaginationProps
+>(function IconOnlyPagination(
+  { label, hasNext, hasPrevious, onNext, onPrevious }: IconOnlyPaginationProps,
+  ref
+) {
+  return (
+    <IconOnlyPaginationStyledNav ref={ref}>
+      <StyledBorderPaginationButton disabled={hasPrevious} onClick={onPrevious}>
+        <Icon name="ri-arrow-m-left-line" />
+      </StyledBorderPaginationButton>
+      {label && (
+        <StyledLabel>
+          <Typography as="div" variant="body1">
```

Comment:
we're using "caption" for the label here and the color should be #2d2d2d `Neutral-B18`
![Uploading image.png…]()

Reply:
removed typography tag as the label type is changed to ReactNode

Reply:
@davidzzz the text color should be neutral-b18, it's still missing from the code